### DISCUSSION TITLE
Mini vMac .dsk disk image support

### DISCUSSION THREAD

**Discussion by nsputnik** [akuker/RASCSI#1166] [Q&A] [answered]
I can't find a disk image file format that I can create or be mountable with PiSCSI that I can also mount in Mini vMac for current MacOS (in my case Intel, OSX 10.13). So far I have tried .hda, .hdr, .hds, .hd1. I can format and mount them all on the hardware Mac but once I download and save them they will not mount them in Mini vMac. Going the opposite way, if I upload and mount a .dsk image in PiSCSI, the hardware Mac will say the disk is unformatted.  Each one will allow me to format the image again but when I get it back to the other platform is is not recognized.  It would be great if PiSCSI could mount these in a way that the real Mac and recognize them.

Mini vMac for works mainly with .dsk files (but I discovered it does not care what the extension of a file is, as long as it was formatted with Mini vMac it will mount the file regardless of what you change the extension to, even removing it altogether). SheepShaver can read .hda created with PiSCSI with no problem (as well as .dsk and .dmg). I know I can use SheepShaver as a bridge for just moving files or change .dsk to .dmg to mount in the host OS but I need to write files directly from a floppy to the PiSCSI image and then mount that in Mini vMac.

**Comment by nsputnik** [comment]
After a little more Googling I found a solution, Jean-Michel Durand's Disk Jockey app for both MacOS and Windows. https://diskjockey.onegeekarmy.eu/ It converts between Mini vMac mountable .dsk to a PiSCSI mountable .hda. It can go back the other way too, converting a .hda mountable PiSCSI back to a to .img which is really a just Mini vMac mountable image (which you can even rename .dsk to keep it consistent and it will still mount in Mini vMac).

**Reply by erichelgeson** [reply, marked as answer]
The reason is vMac disks are only HFS volumes, so they do not have a partition map or scsi driver partition like real hard disk drives do. DJ is a good option to convert back and forth.
